util/mk-fipsmodule-cnf.pl

```diff
@@ -69,6 +70,7 @@
 tdes-encrypt-disabled = $tdes_encrypt_disabled
 rsa-sign-x931-pad-disabled = $rsa_sign_x931_pad_disabled
 hkdf-key-check = $kdf_key_check
+kmac-key-check = $mac_key_check
```

Comment:
not sure this is great placement amongst the KDFs.

Reply:
trying to avoid conflicts for now.. will reorder all of them later